getting-started/modules/core/valtimo-dependency-versions.md

```diff
@@ -0,0 +1,55 @@
+# Valtimo dependencies
+
+## Dependencies
+
+The `valtimo-dependency-versions` module specifies dependency versions for Valtimo.
+When this module is used, please specify dependency versions only when needed. 
+
+> NOTE: This dependency will not add any other dependencies to the classpath. It will only define the versions.
```

Comment:
```suggestion
> **_NOTE:_** This dependency does not add any other dependencies to the classpath. It only defines the versions.
```

Partially suggesting this so it is in line with `/introduction/modules/modules.md`